samples/cubDeviceFind/README.md

```
@@ -0,0 +1,52 @@
+### [CUDA Runtime API](http://docs.nvidia.com/cuda/cuda-runtime-api/index.html)
+
+cudaDeviceSynchronize, cudaGetDeviceProperties
+
+## Dependencies needed to build/run
+
+[CCCL 3.3+](https://github.com/NVIDIA/cccl). Fetched automatically via CPM at configure time (pinned to `v3.3.3`). Override with `-DCCCL_SOURCE_DIR=/path/to/cccl` to use a local checkout.
+
+## Prerequisites
+
+Download and install the [CUDA Toolkit](https://developer.nvidia.com/cuda-downloads) for your corresponding platform.
+Make sure the dependencies mentioned in [Dependencies]() section above are installed.
```

markdownlint-cli2 (0.22.1)

[warning] No empty links

(MD042, no-empty-links)